modules/extensions-support/src/main/java/org/mule/module/extension/internal/util/CapabilityUtils.java

```diff
@@ -85,7 +86,17 @@ public static boolean isCapableOf(Set<?> capabilities, Class<?> capabilityType)
 
     }
 
-    public static <T> T getSingleCapability(Set<?> capabilities, Class<T> capabilityType) {
+    /**
+     * Returns the first item in the {@code capabilities} {@link Set} which is an
```

Comment:
Can you update this javadoc to be more like the one from the overloaded version of it? ( Expects {@code capabilities} to have at most one capability of type {@code capabilityType} and returns such capability.)